russian/dance_lezginka/cake_bomb/cake_bomb.md

```diff
@@ -0,0 +1,9 @@
+Вам не нравится, что эти лица неопределенной национальности продолжают вторгаться в Ваш зефирный дом.
+
+Вы вспоминаете, что у Вас в кармане завалялась парочка кексовых бомб. Вы достаете бомбы и делаете предупреждения, но никто не собирается уходить.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.